EESSI-install-software.sh

```diff
@@ -189,19 +189,18 @@ pr_diff=$(ls [0-9]*.diff | head -1)
 
 # install any additional required scripts
 # order is important: these are needed to install a full CUDA SDK in host_injections
-install_scripts_changed=$(cat ${pr_diff} | grep '^+++' | cut -f2 -d' ' | sed 's@^[a-z]/@@g' | grep '^install_scripts.sh$' > /dev/null; echo $?)
-if [ ${install_scripts_changed} == '0' ]; then
-    # for now, this just reinstalls all scripts. Note the most elegant, but works
-    ${TOPDIR}/install_scripts.sh --prefix ${EESSI_CVMFS_REPO}
-fi
+# for now, this just reinstalls all scripts. Note the most elegant, but works
+${TOPDIR}/install_scripts.sh --prefix ${EESSI_PREFIX}
```

Comment:
@boegel this is intended to install the scripts into CVMFS, you mean that this is not actually working?

Reply:
It will install scripts in `/cvmfs/software.eessi.io` in the build environment that the bot sets up, but that's not the same as actually deploying those scripts in the EESSI repository on the Stratum 0.

Only stuff that is included in the tarball (which is created with the 🥁 `create_tarball.sh` script in this repo) is actually ingested in the EESSI repo on deployment.
Stuff in `versions/2023.06/scripts/` does not get included in the tarball, currently, so it doesn't get deployed.

Reply:
Ok, but the intention here was to actually do the install, otherwise there is no reason for the installation step

Reply:
I'll fix it this evening then

Reply:
Note that https://github.com/EESSI/filesystem-layer/blob/main/scripts/ingest-tarball.sh also may or may not work for this. It _seems_ that it was anticipated that a `scripts` directory would be installed, so probably as long as the top-level dir within the repo is named _exactly_ that, I think it'll work. But it's good to be aware that there could be issues here. In this scenario, I believe it _would_ end up in the tarball, but would still not be ingested by the stratum 0. So in case you encounter that pattern, this is where script which likely should be fixed.